src/containers/Review/ReviewSubmit.tsx

```diff
@@ -98,7 +127,8 @@ const ReviewSubmitButton: React.FC<ReviewSubmitProps & ReviewSubmitButtonProps>
   })
 
   const { ConfirmModal, showModal: showConfirmModal } = useConfirmationModal({
-    ...messages.REVIEW_SUBMISSION_CONFIRM,
+    onConfirm: () => submission(),
+    showCancel: false,
```

Comment:
Can we keep the cancel button please? It's very strange not to have an obvious "Cancel" option.

Reply:
Sure! I'll do that now